### DISCUSSION TITLE
Accessing  authenticationKey

### DISCUSSION THREAD

**Discussion by fstof** [serverpod/serverpod#2326] [Q&A] [answered]
So since upgrading to v2.0.0 we lost access to `session.authenticationKey` from inside an endpoint

This is required when we do any upstream api calls that might be required.

How would I go about getting the key? Or is this actually a bug?

**Comment by SandPod** [comment, marked as answer]
Hey @fstof!

It appears that the getter for the authentication key was accidentally removed from the session object.
We will create a fix for this ASAP.

**Reply by SandPod** [reply]
A fix for this has been merged and will be available in the next release of Serverpod.
